pubsub/gcp/config.go

```diff
@@ -10,6 +10,8 @@ import (
 type Config struct {
 	gcp.Config
 
+	// Project ID in the GCP Console
+	ProjectID string `envconfig:"GCP_PROJECT_ID"`
```

Comment:
There should already be a ProjectID attribute in the embedded `gcp.Config` struct so we don't need to add one here.